ga4gh/datamodel/reads.py

```diff
@@ -182,19 +252,15 @@ def toProtocolElement(self):
         readGroup.referenceSetId = None
         readGroup.sampleId = None
         stats = protocol.ReadStats()
-        stats.alignedReadCount = self.getNumAlignedReads()
-        stats.unalignedReadCount = self.getNumUnalignedReads()
+        stats.alignedReadCount = \
+            self._parentContainer.getNumAlignedReads()
```

Comment:
I think we'll have to put -1 in here for now, and look at generating some metadata JSON files like we do (or, soon will do) for references. There's really no other way to do it.